lib/ansible/modules/windows/win_updates.py

```diff
@@ -33,22 +33,11 @@
         version_added: '2.5'
     category_names:
         description:
-        - A scalar or list of categories to install updates from
+        - A scalar or list of categories to install updates from. To get the list
+          of categories, run the module with C(state=query). The category must
+          be the full category string, but is case insensitive.
         type: list
         default: [ CriticalUpdates, SecurityUpdates, UpdateRollups ]
-        choices:
```

Comment:
I wouldn't list it in the choices but I would still reference the ones below (maybe the proper names with spaces) in a description entry.